db/schema.rb

```diff
@@ -22,6 +22,8 @@
     t.json     "link_errors"
     t.datetime "created_at",    null: false
     t.datetime "updated_at",    null: false
+    t.integer  "link_id"
+    t.index ["link_id"], name: "index_checks_on_link_id", using: :btree
```

Comment:
Ah because I started adding the migration here before I realised the collision with AR `errors`, so the migration here is older than the one in https://github.com/alphagov/link-checker/pull/6 which shouldn't matter once both are merged should it? I've confused myself now.